index.html

```diff
@@ -23,7 +23,7 @@ <h1 id="what-this-is">What This Is</h1>
 <p>This fork can be considered a supported version of <a href="https://github.com/openssl/openssl/pull/8797">OpenSSL PR 8797</a>. We will endeavor to track OpenSSL releases within a day or so, and there is an item below about how we'll follow their tagging.</p>
 <p>On to the questions and answers.</p>
 <h2 id="what-about-branches">What about branches?</h2>
-<p>We don't want to conflict with OpenSSL branch names. Our current plan is to append <code>+quic</code>. Release tags are likely to be the QUIC branch with <code>-releaseX</code> appended. For example, the OpenSSL tag <code>openssl-3.0.0-alpha12</code> would have a branch named <code>openssl-3.0.0-alpha12+quic</code> and a release tag of <code>openssl-3.0.0-alpha12+quic-release1</code></p>
+<p>We don't want to conflict with OpenSSL branch names. Our current plan is to append <code>-quicX</code> where X is the iteration number. To get the code use git to pull down the tag.
```

Comment:
If you want to ask for volunteers to make releases, perhaps here.  Or don't bother.